PDFtoWord/main.py

```diff
@@ -0,0 +1,9 @@
+from pdf2docx import Converter
+
+pdf_file = 'sample.pdf'
```

Comment:
take the input for pdf file path for user

Reply:
Okay